src/staking.rs

```diff
@@ -44,16 +44,16 @@ impl Client {
         Self { base_url, client }
     }
 
-    /// Fetch the public key of the staking server
-    pub fn address(&self) -> Result<PubKeyBin> {
-        let request_url = format!("{}/address", self.base_url);
+    /// Fetch the public maker key for a given gateway key
```

Comment:
Perhaps this should read "for a given **onboarding** key".

Reply:
Sorry, merged before you left that comment.. Fixed on master